climada/hazard/tc_tracks.py

```diff
@@ -387,36 +405,52 @@ def read_ibtracs_netcdf(self, provider=None, rescale_windspeeds=True, storm_id=N
         if estimate_missing and not rescale_windspeeds:
             LOGGER.warning(
                 "Using `estimate_missing` without `rescale_windspeeds` is strongly discouraged!")
-        self.data = list()
-        fn_nc = SYSTEM_DIR.joinpath(file_name)
-        if not fn_nc.is_file():
+
+        ibtracs_path = SYSTEM_DIR.joinpath(file_name)
+        if not ibtracs_path.is_file():
             try:
                 download_ftp(f'{IBTRACS_URL}/{IBTRACS_FILE}', IBTRACS_FILE)
-                shutil.move(IBTRACS_FILE, fn_nc)
+                shutil.move(IBTRACS_FILE, ibtracs_path)
             except ValueError as err:
                 raise ValueError(
                     f'Error while downloading {IBTRACS_URL}. Try to download it manually and '
-                    f'put the file in {SYSTEM_DIR}') from err
+                    f'put the file in {ibtracs_path}') from err
```

Comment:
I am not 100%, but I thought remembering that one should write the message as

```suggestion
                    'put the file in %s', ibtracs_path) from err
```

Can you confirm @tovogt and/or @emanuel-schmid ?

Reply:
I think you confused this with the `logging` functions. For logging-functions it might be undesirable to format the string in case the log-level will cause the result to be discarded anyways: https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/ But this is not the case here.

Reply:
Thanks for the reminder, I indeed confused both of them.